products/workers/src/content/platform/limits.md

```diff
@@ -312,6 +288,8 @@ Durable Objects are currently in beta. See [their docs](/learning/using-durable-
 
 - Storage values of up to 32 KiB (32768 bytes)
 
-- 500ms of CPU time per request, including websocket messages (this will be raised to match the Unbound limit after the beta period)
+- 30s of CPU time per request, including websocket messages
+
+Durable Objects scale well across Objects, but are inherently single-threaded.  A baseline of 100 req/sec is a good floor estimate of the request rate an individual Object can handle, though this will vary with workload. 
```

Comment:
```suggestion
Durable Objects scale well across Objects, but each object is inherently single-threaded.  A baseline of 100 req/sec is a good floor estimate of the request rate an individual Object can handle, though this will vary with workload. 
```